tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -73,24 +76,33 @@ public function testCreateNews() {
 
     // Create a News item.
     $this->drupalGet('node/add/oe_sc_news');
-    $page = $this->getSession()->getPage();
     $page->fillField('Title', 'Example title');
     $page->fillField('Content', 'Example Content');
     $page->fillField('Introduction', 'Example Introduction');
-    $page->fillField('Date', '2022-01-24');
     $media_name = $media_image->getName() . ' (' . $media_image->id() . ')';
     $page->fillField('Media item', $media_name);
     $page->fillField('Caption', 'Starter Image caption');
     $page->pressButton('Save');
 
-    // Assert media document has been created.
+    // Assert media document has been created and publication date
+    // is current date because it's not fulfilled.
     $assert_session->pageTextContains('News Example title has been created.');
     $assert_session->pageTextContains('Example title');
     $assert_session->pageTextContains('Example Content');
     $assert_session->pageTextContains('Example Introduction');
-    $assert_session->pageTextContains('01/24/2022');
+    $assert_session->pageTextContains(DrupalDateTime::createFromTimestamp(
+      \Drupal::time()->getRequestTime(),
+      DateTimeItemInterface::STORAGE_TIMEZONE
+    )->format('m/d/Y'));
```

Comment:
I still don't trust this part, it could lead to unexpected fail if the test runs at the wrong time.
Either we test this in a more predictable way, or we test only that the field is not empty.
See also my older comment which is now outdated.

Reply:
What you could do:
- select the only `<time>` tag on the page
- use regex to find the `\d+/\d+/\d+` date string in the element text. If you are lazy, you are finished now.
- (don't) If you are more ambitious, you could assert that the date is within 1d range of the current date. This gets rid of possible mistakes. But I think we should not do it, too many moving parts, too much conditional logic.
- (don't) If you are really ambitious you could use https://www.drupal.org/project/datetime_testing and simulate creating a news half hour before midnight and half hour after midnight, or just 1x at noon. we do that in project for the progress bar. But imo we should not spend that extra time + effort.

I leave the last two points for the next reviewer so they can follow the decision process and share their opinion.

Reply:
Btw, if we would use 'classy' theme instead of 'stark', we could select the field using a class.
But it is not really needed here, seems like there is only one `<time>` tag.